pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -1204,3 +1205,36 @@ func (r *remoteRuntimeService) CheckpointContainer(options *runtimeapi.Checkpoin
 
 	return nil
 }
+
+func (r *remoteRuntimeService) GetContainerEvents(containerEventsCh chan *runtimeapi.ContainerEventResponse) error {
+	errCh := make(chan error)
+
+	if r.useV1API() {
```

Comment:
nit, if you inverse this and return early, then the rest of the code could have shallower indentation